instrumentation/mybatis/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mybatis/MybatisInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Arrays;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class MybatisInstrumentationModule extends InstrumentationModule {
+
+  public MybatisInstrumentationModule() {
+    super("mybatis");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return Arrays.asList(new MybatisExecuteInstrumentation());
```

Comment:
usually we use `Collections.singletonList` when we need a list with only one element

Reply:
> usually we use `Collections.singletonList` when we need a list with only one element

I adjusted it.